src/integrationTests/communities.test.ts

```diff
@@ -0,0 +1,218 @@
+import { Crypto } from '@peculiar/webcrypto';
+import {
+  createCommunity,
+  joinCommunity,
+  getCommunityOwnerData,
+  tryToJoinOfflineRegistrar,
+  assertReceivedCertificates,
+  assertReceivedChannelsAndSubscribe,assertReceivedMessages, launchCommunitiesOnStartupSaga, sendMessage
+} from './communitiesTestUtils';
+import {
+  createUserCsr,
+  UserCsr,
+} from '@zbayapp/identity/lib/requestCertificate';
+import { StoreKeys } from '../sagas/store.keys';
+import { identityAdapter } from '../sagas/identity/identity.adapter';
+import config from '@zbayapp/identity/lib/config';
+import { createApp, createAppWithoutTor } from './utils';
+import { communitiesAdapter } from '../sagas/communities/communities.adapter';
+import {
+  CommunitiesState,
+  Community,
+} from '../sagas/communities/communities.slice';
+import {
+  Identity,
+  IdentityState,
+} from '../sagas/identity/identity.slice';
+
+import {
+  CommunityChannels,
+  PublicChannelsState,
+} from '../sagas/publicChannels/publicChannels.slice';
+import { channelsByCommunityAdapter } from '../sagas/publicChannels/publicChannels.adapter';
+
+
+jest.setTimeout(600_000);
+
+const crypto = new Crypto();
+
+global.crypto = crypto;
+
+describe('test creating, joining and relaunching communities, sending messages, replicate channels, messages and certificates', () => {
```

Comment:
This naming seems non-optimal, since the heading and the test itself is the same name, which is mostly redundant, and since the name is really long and includes a bunch of things. Don't we want separate assertions for each step?